geth/jail/internal/loop/loop.go

```diff
@@ -46,7 +46,8 @@ type Loop struct {
 	lock      sync.RWMutex
 	tasks     map[int64]Task
 	ready     chan Task
-	accepting bool
+	closer    sync.Once
+	closeChan chan struct{}
```

Comment:
Maybe `closedChan` or `closedCh` (adjective) as it's usually used to check if the loop is closed. Similarly to a pattern: `done := make(chan struct{})`.